src/python/genny/genny_auto_tasks.py

```diff
@@ -204,6 +225,34 @@ def validate_user_workloads(workloads):
     return errors
 
 
+def get_prepare_environment_vars(task_name, fname):
+    """
+    :param task_name: A string representing this task, determined from filename.
+    :param fname: A filename for this workload.
+    :return: A list of environment var dictionaries, one for each mongodb_setup used.
+    """
+    autorun_spec = None
+    prepare_environment_vars_template = {
+        'test': task_name,
+        'auto_workload_path': fname
+    }
+
+    full_filename = '{}/src/workloads/{}'.format(get_project_root(), fname)
+    with open(full_filename, 'r') as handle:
+        try:
+            workload_dict = yaml.safe_load(handle)
+            autorun_spec = AutoRunSpec.create_from_workload_yaml(workload_dict)
+        except Exception as e:
+            pass
```

Comment:
this is too broad an exception message since it can catch programmer error. What exception are you expecting may be thrown here and why?

Reply:
I wasn't sure; it was just already there when I refactored to extract this function. Looks like deleting the handling causes the yaml to raise a ComposerError for duplicate anchors in `scale/UniqueIndexStress.yml` specifically. I was able to remove the duplicate anchor declarations and that fixed it.